### ISSUE TITLE
In widget component reference, include a (preferrably annotated) screenshot

### ISSUE THREAD

**Issue by SRGDamia1** [openhab/openhab-docs#1800]
It would be very helpful to see what the widget blocks look like in skeleton.

**Comment by Confectrician** [comment]
Do you refer to https://www.openhab.org/docs/tutorial/auto_overview.html ?

**Comment by SRGDamia1** [comment]
Yes. It's very good documentation!  It would just be helpful in the reference sheets to have an image.  So, for example on the page for an `oh-knob` (https://www.openhab.org/docs/ui/components/oh-knob.html) have a picture of what a knob looks like.
